dotnet/src/Functions/Functions.Yaml/PromptExecutionSettingsTypeConverter.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using YamlDotNet.Core;
+using YamlDotNet.Core.Events;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.BufferedDeserialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Allows custom deserialization for <see cref="PromptExecutionSettings"/> from YAML prompts.
+/// </summary>
+internal sealed class PromptExecutionSettingsTypeConverter : IYamlTypeConverter
+{
+    /// <inheritdoc/>
+    public bool Accepts(Type type)
```

Comment:
Good case for `static`

```suggestion
    public static bool Accepts(Type type)
```

Reply:
The method is declared in the `IYamlTypeConverter` interface and should be implemented by the class as and instance one.